### ISSUE TITLE
include `cppcheck-gui` in package

### ISSUE THREAD

**Issue by firewave** [iosifache/cppcheck-snap#3]
It might make sense to offer the `cppcheck-gui` application in the package as well.

**Comment by firewave** [comment]
Turns out there is already a separate package for this which is also extremely outdated: https://snapcraft.io/cppcheckgui.

**Comment by iosifache** [comment]
Yes, this is why [I also proposed contacting the maintainer](https://sourceforge.net/p/cppcheck/discussion/development/thread/04c7898722/#ec28). I hope adding `cppcheck-gui` in this snap will not confuse the users.

**Comment by firewave** [comment]
> I hope adding `cppcheck-gui` in this snap will not confuse the users.

If there is an existing Snap for that it should probably stay that way. Also makes sense to have the CLI and GUI separate as they do not depend on each other. That would be in line with the more common packaging on *nix systems nowadays.

**Comment by iosifache** [comment]
@firewave, please allow me to contact the maintainer of that snap. Perhaps they can start publishing new releases again. Although we can consider this issue closed since we will no longer embed `cppcheck-gui` in this snap, I will leave this issue open to share updates with you.
